### ISSUE TITLE
Document everything!!!

### ISSUE THREAD

**Issue by mitar** [cloyne/network#88]
* servers
* Docker
* how we allocate IP ranges
* new semester things (link to the document)
* APs
* firmware
* connections with other houses
* uplink

**Comment by jmrtns** [comment]
How to modify DNS should also be on here, I'm fine with using SQL from CLI but I should probably deploy a web based SQLite manager or move the DNS to pgsql or mysql. I'll also look into a web interface specifically for powerdns.

**Comment by mitar** [comment]
What SQL? What are you talking about? DNS is stored in GitHub.

**Comment by mitar** [comment]
https://github.com/cloyne/docker-powerdns-master/tree/master/etc/powerdns/bind

**Comment by jmrtns** [comment]
I wanted to add some DNS entries, I dug through that but didn't understand it much, our powerdns docker image uses SQLite and the powerdns guides I could find online pointed towards using SQL to add entries.

**Comment by mitar** [comment]
No, our PowerDNS docker image does not use SQLite. It uses those files. You edit them, image gets rebuild on Docker Hub, you redeploy them, and DNS is updated.

Those are [zone files](https://en.wikipedia.org/wiki/Zone_file).

**Comment by mitar** [comment]
I added more to the documentation. About mailing lists, DNS servers, web site, Docker images, Salt.

**Comment by clonm** [comment]
I made a diagram of how all the docker dependencies fit together: ![cloyne network diagram-docker 1](https://user-images.githubusercontent.com/6488743/51475679-f6a62080-1d37-11e9-81e9-d223c0d32e2c.png) @kngnm @mitar
